concordia/models.py

```diff
@@ -362,6 +362,7 @@ class SimpleContentBlock(models.Model):
     updated_on = models.DateTimeField(editable=False, auto_now=True)
 
     label = models.CharField(
+        unique=True,
```

Comment:
@rstorey this raised one question: should this just be a `SlugField`?

Reply:
Yes that's a good idea

Reply:
This is done and I squashed the migrations since they've never lived outside of this branch